modules/videoModule/videoImpressionVerifier.js

```diff
@@ -65,7 +66,9 @@ export function videoImpressionVerifier(vastXmlEditor_, bidTracker_) {
 
     if (vastUrl) {
       const url = new URL(vastUrl);
-      url.searchParams.append(UUID_MARKER, uuid);
+      if (!config.getConfig('cache.useLocal')) {
+        url.searchParams.append(UUID_MARKER, uuid);
+      }
```

Comment:
why wouldn't we append the UUID Marker when the url is local ? The idea is to know that the ad from the bid actually rendered. This is most importantly for when the bid competes in the ad server auction, but without a marker we'll never if the ad from the bid actually rendered.

Reply:
I don't think that local Blob URLs can be modified in any way, adding query parameters to it will make it no longer point to the current blob and return 404. We might think of some alternative for handling this tracking in case of using local cache